articles/templates/about-template-output/about-template-output.md

```diff
@@ -0,0 +1,47 @@
+---
+uid: templates.about-template-output
+---
+# About Template Output
+
+Templates are used to produce generated content and to write them to files. Where these files are located can be determined by the `Output Configuration Designer`.
+
+![Output Config Template Output](images/output-config-template-output-side-by-side.png)
+_Example of Output Configuration Layouts_
+
+When Modules are installed that contain Text Templates, they will automatically install output destination items for those Templates called `Template Output` items. In the examples above you will notice that these `Template Output` items are located inside folders or SDK projects (for example, Visual Studio's SDK offers C# Projects as a way to organize files). You will also notice that these `Template Output` items are named according to their respective Template-Identifiers which are found in the Modules that are installed. This gives visibility to the user to know which Templates are present and where they are destined to produce file output during a Software Factory Execution.
+
+## Unassigned Template Output
+
+It may be possible that during Module development that a Template does not have a predefined `Role` (we will cover this in the next section) or the `Role` does not exist inside the `Output Configuration Designer`. This will cause the Template to be initially placed in the root level of your `Output Configuration Designer`.
+
+![Unassigned Template in Output Configuration Designer](images/output-config-vs-unassigned-template.png)
+
+> [!WARNING]
+> Unassigned `Template Outputs` may cause warnings or errors during a Software Factory Execution, so ensure that they are relocated to the expected output location in the `Output Configuration Designer`.
```

Comment:
To me, "It may be possible..." feels a bit too indirect, or superfluous, we should just be doing the most straight forward expressing of information possible. Also keep in mind that unassigned template outputs is a problem in the VS designer, but not the "folders" designer (although double check this), so I don't think the `[!WARNING]` box is appropriate here.

I'm thinking this sections should be something like:

> Some designers (such as the Visual Studio designer) will show unassigned Template Outputs in red:
>
> (insert screenshot here)
>
> And running the the SF while these template are unassigned will result in errors during the SF execution about there being unassigned templates.
>
> Templates can specify where there "Template Output" should be placed by default during module installation so that things work automatically, see the _(link to section)_ for more details.

Reply:
In the "(link to section)" bit, are you referring to the Roles and Default locations the proceed that paragraph?